hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestCreateTable.scala

```diff
@@ -505,4 +505,29 @@ class TestCreateTable extends TestHoodieSqlBase {
     }
   }
 
+  test("Test Create Table Exists In Catalog") {
+    val tableName = generateTableName
+    spark.sql(
+      s"""
+         |create table $tableName (
+         | id int,
+         | name string,
+         | price double
+         |) using hudi
+         |""".stripMargin
+    )
+
+    spark.sql(s"alter table $tableName add columns(ts bigint)")
+
+    // Check "create table if not exist" works after schema evolution.
```

Comment:
basically this create table statement will be ignored right? so, old table schema remains.

For eg:
if original table had 3 fields. 
and then alter table to add 4th field.
again, create table if not exists with 5 fields.

Is the final state of table is just 4 fields ?

Reply:
Yes, the create table if not exists will be ignored if the table has exists in the catalog.